dashboard/config/puma.rb

```diff
@@ -7,18 +7,23 @@
 else
   bind "tcp://#{CDO.dashboard_host}:#{CDO.dashboard_port}"
 end
+
 workers CDO.dashboard_workers
 threads 1, 5
 
-drain_on_shutdown
+directory deploy_dir('dashboard')
 
-# nginx already buffers/queues requests so disable Puma's own queue.
-queue_requests false
+unless CDO.rack_env == :development
```

Comment:
```suggestion
unless CDO.rack_env?(:development)
```
https://github.com/code-dot-org/code-dot-org/blob/77258a3e359d2e375130fd79a97c82103a0d0c94/lib/cdo.rb#L315-L317